package.json

```diff
@@ -80,11 +80,11 @@
     "@aws-sdk/client-dynamodb": "3.799.0",
     "@aws-sdk/client-ssm": "3.799.0",
     "@govuk-one-login/data-vocab": "1.9.2",
-    "@govuk-one-login/frontend-analytics": "3.3.0",
-    "@govuk-one-login/frontend-device-intelligence": "1.1.0",
+    "@govuk-one-login/frontend-analytics": "4.0.0",
```

Comment:
@AmritSidhu it sounds like there's a breaking change in the move to 4, have you checked [frontend-analytics ](https://github.com/govuk-one-login/govuk-one-login-frontend/tree/main/packages/frontend-analytics#upgrading-to-v4) ?